extensions/panache/panache-hibernate-common/runtime/src/main/java/io/quarkus/panache/hibernate/common/runtime/PanacheJpaUtil.java

```diff
@@ -66,11 +66,6 @@ public static String createFindQuery(Class<?> entityClass, String query, int par
             return "FROM " + getEntityName(entityClass);
         }
 
-        if (isNamedQuery(query)) {
-            // we return named query as is
-            return query;
-        }
```

Comment:
I understand the other changes but could you clarify that one?

Reply:
@gsmet all code path that call this method already does this check, some even does it again later but I didn't modify them to be conservative.
This is an internal method and we have tests for all of these so it should be safe.